src/Mod/TechDraw/App/AppTechDrawPy.cpp

```diff
@@ -43,6 +43,9 @@
 #include <Base/PyWrapParseTupleAndKeywords.h>
 #include <Base/Vector3D.h>
 #include <Base/VectorPy.h>
+
+#include <Gui/Application.h>

```

Comment:
Including GUI in App looks strange

Reply:
Yes,  it does.  Using the QApplication based test suggested by @hyarion would avoid that.